pytest_allure_dsl.py

```diff
@@ -198,6 +204,16 @@ def _yaml_load(self, string):
             self.description_load_error = e
             return {}
 
+    def _check_steps_was_used(self):
+        not_used_steps = []
+
+        for key in self.steps.keys():
+            if key not in self._steps_was_used:
+                not_used_steps.append(key)
+
+        if not_used_steps:
+            raise StepsWasNotUsed(*not_used_steps)
```

Comment:
I think it is helpful to describe some actions made by fixtures in the description of a test. 
So, I think we should afford to have steps described, but not implemented.

Reply:
We can extand DSL as we wish for this
```
"""
steps:
  1:
    title: some title of implemented step
    not-implemented-steps:
      1.1: some title of not implemented step 
""" 
````
but if code will changed and doc string not, we don't know about this